### ISSUE TITLE
Update: Modifying pre_get_posts for Categories & Tags to show Custom Post Types

### ISSUE THREAD

**Issue by jimtrue** [pods-framework/pods-docs#90]
## Issue Info
* Docs URL:  https://docs.pods.io/code-snippets/modifying-pre_get_posts-categories-tags-show-custom-post-types/
* Doc Title: Modifying pre_get_posts for Categories & Tags to show Custom Post Types
* Issue Type: Update

## Issue Description
I think this only worked if the search term was BOTH a category and a tag?  I tweaked it to:

if (  !is_admin() // Only target front end queries
          && $q->is_main_query()
          && ( $q->is_category()
          || $q->is_tag()    )   
    )

and that seemed to work?

## Acceptance Criteria

**This doc is ready to publish when:**
- [ ] Overview Created
- [ ] Quick Start (if Needed) Created
- [ ] Excerpt Created for Summary
- [ ] Keywords, Difficulty and Context completed
- [ ] Related Docs, Videos, FAQs, Code Snippets referenced
- [ ] Code Review Completed (if needed)

From: Kelly Dorset kellydorset+pods@googlemail.com

**Comment by quasel** [comment]
Yeah if you want both `||` or remove one and/or add more examples - it's kinda explained in the text but nt very clear/optimal ^^
